logging/OpenShift.md

```diff
@@ -46,22 +46,64 @@ oc login [cluster-hostname] -u [userID]
 ```
 
 ## <a name="l_os_cust"></a>Customization 
+#### General Customization
 
 Customization of SAS Viya Logging on OpenShift deployment follows the same process as in a standard logging deployment, which uses the USER_DIR environment variable to specify the location of your customization files. See the [logging README](../logging/README.md#log_custom) for information about the customization process.
 
-To change the number of days that log messages from OpenShift infrastructure components 
-are retained, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable. The default 
-value is `1` (1 day).
+### OpenShift-Specific Customization
 
-No customizations are required, even if you are using ingress, because the `deploy_logging_open_openshift.sh` script defines a route for Kibana.
+#### OpenShift Infrastructure Message Retention
+You can change the number of days that log messages from OpenShift infrastructure namespaces (any namespace that starts with "openshift") 
+are retained so that the messages do not fill up the storage space. To change the retention period, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable. The default value is `1` (1 day).
 
+#### Access Using Route Objects
 OpenShift uses [route](https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html) objects, a feature unique to OpenShift, to access Kibana and (optionally) the Elasticsearch API endpoint. This makes it unnecessary to configure ingress objects or surface nodePorts.
+No customizations are required, even if you are using ingress, because the `deploy_logging_open_openshift.sh` script defines a route for Kibana.
+
+#### TLS
+OpenShift route objects are always TLS-enabled. Traffic is encrypted between the user 
+and OpenShift and again within the OpenShift cluster. The TLS certificates used on 
```

Comment:
I'm fine with keeping the explanatory text for others in place but I think we should use the OpenShift terminology (e.g. routes are configured with reencrypt termination).   OpenShift admins will be looking for that kind of terminology.

Reply:
added mention of re-encrypt termination